bindings/python/CMakeLists.txt

```diff
@@ -148,6 +148,11 @@ FUNCTION (PY_ADD_PACKAGE_DIRECTORY NAME)
     SET (SETUP_PY "${CMAKE_CURRENT_BINARY_DIR}/${NAME}/setup.py")
     CONFIGURE_FILE (${SETUP_PY_IN} ${SETUP_PY})
 
+    # Configure MANIFEST.in file
```

Comment:
@lucas-bremond Any cons into adding these files in the source distribution? size of the package? or control of the version? 🤔